dragula.js

```diff
@@ -58,8 +59,13 @@ function dragula (initialContainers, options) {
 
   return drake;
 
-  function isContainer (el) {
-    return drake.containers.indexOf(el) !== -1 || o.isContainer(el);
+  function isContainer (el, handle) {
+    handle = handle || null;
+    if (handle && o.allowNestedContainers) {
+       return drake.containers.indexOf(el) !== -1 && o.isContainer(el, handle);
```

Comment:
Are you completely sure, that this is the right way? I tried your approach and got some weird results. This requires all containers to be registered within Drake object, effectively making it hard to build a nested structure.
This would do totally fine:
```
function isContainer(el, handle) {
       return drake.containers.indexOf(el) !== -1 || o.isContainer(el, handle);
}
```